ztp/kube-compare-reference/Makefile

```diff
@@ -24,4 +27,32 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1
```

Comment:
If the `helm` step below installs the helm executable to the local directory, perhaps we should also remove it as part of the `clean` step?